.clang-format

```diff
@@ -39,11 +39,13 @@ ContinuationIndentWidth: 4
 IndentCaseLabels: true
 IndentPPDirectives: None
 IndentWidth: 4
+InsertBraces: true
```

Comment:
InsertBraces is problematic for clang-format-12 & Qt Creator.  Works nicely with 15.  Do we have a minimum version requirement for clang-format?